config/config.toml.example

```diff
@@ -37,6 +37,9 @@ mem-quota-query = 34359738368
 # Set to true to enable use of temporary disk for some executors when mem-quota-query is exceeded.
 oom-use-tmp-storage = true
 
+# The location of temporary disk storage using by some executors when mem-quota-query is exceeded.
```

Comment:
Please document the default value like `it defaults to <TMPDIR>/tidb/tmp-storage if it is unset`.